src/bijectors/coupling.jl

```diff
@@ -118,6 +118,20 @@ Combines `x_1`, `x_2`, and `x_3` into a single vector.
 """
 @inline combine(m::PartitionMask, x_1, x_2, x_3) = m.A_1 * x_1 .+ m.A_2 * x_2 .+ m.A_3 * x_3
 
+function ChainRulesCore.rrule(::typeof(combine), m::PartitionMask, x_1, x_2, x_3)
```

Comment:
Can you move the rule to src/chainrules.jl and add a test in test/ad/chainrules.jl?

Reply:
Will do

Reply:
Done.